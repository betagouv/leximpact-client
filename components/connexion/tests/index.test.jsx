import { createShallow } from "@material-ui/core/test-utils";

import LoginForm from "../index";

describe("components | connexion | index", () => {
  let shallow;
  beforeAll(() => {
    shallow = createShallow();
  });

  describe("snapshot", () => {
    it("doit correspondre avec les props requises", () => {
      const props = {};
      const wrapper = shallow(<LoginForm {...props} />);
      expect(wrapper).toBeDefined();
      expect(wrapper).toMatchSnapshot();
    });
  });
});
