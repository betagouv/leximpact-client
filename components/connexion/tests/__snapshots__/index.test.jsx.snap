// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | connexion | index snapshot doit correspondre avec les props requises 1`] = `
<Connexion
  classes={
    Object {
      "avatar": "Connexion-avatar-1",
      "avatarContainer": "Connexion-avatarContainer-2",
      "avatarIcon": "Connexion-avatarIcon-3",
      "form": "Connexion-form-4",
      "spanBolder": "Connexion-spanBolder-5",
      "spanLeximpactBolder": "Connexion-spanLeximpactBolder-6",
      "spanLeximpactLigther": "Connexion-spanLeximpactLigther-7",
      "title": "Connexion-title-8",
    }
  }
/>
`;
